lib/lz4frame.c

```diff
@@ -46,11 +46,25 @@ You can contact the author at :
 #endif
 
 
+/*-************************************
+*  Tuning parameters
+**************************************/
+/*
+ * LZ4_HEAPMODE :
+ * Select how default compression functions will allocate memory for their hash table,
+ * in memory stack (0:default, fastest), or in memory heap (1:requires malloc()).
+ */
+#ifndef LZ4_HEAPMODE
+#  define LZ4_HEAPMODE 0
```

Comment:
Good point.
There is one small advantage for keeping the name `LZ4_HEAPMODE`  : 
It becomes possible to compile with `-DLZ4_HEAPMODE=1`, and it's applicable to the entire `lz4` library / binary.
Otherwise, one would need to set both `-DLZ4_HEAPMODE=1 -DLZ4F_HEAPMODE=1`

Reply:
I slightly prefer `LZ4F_`, since thats what everything in the API is prefixed with, but I don't have a strong preference.

Reply:
OK, there is a narrow advantage to `LZ4F_`, due to consistency.
So let's switch to `LZ4F_HEAPMODE`.